config/luci_view.rviz

```diff
@@ -0,0 +1,195 @@
+Panels:
+  - Class: rviz_common/Displays
```

Comment:
I see why this is here to allow the launch file below to access it but is there anything from this that should be upstream to the generic one we provide here https://github.com/lucimobility/luci-ros2-sdk/tree/main/rviz

Reply:
The only real difference in this launch file is that we have the robot description topic already subscribed, which can be added upstream

Reply:
Okay, I dont think its a huge deal to pull the rivz changes up to the main one as the URDF stuff is not a default currently in the SDK. Im fine with how this is then. If it becomes used more often then we can upstream the RVIZ changes.